inbm/dispatcher-agent/dispatcher/sota/snapshot.py

```diff
@@ -38,14 +38,16 @@ class Snapshot(metaclass=ABCMeta):  # pragma: no cover
     @param sota_cmd: SOTA command (update)
     @param snap_num: snapshot number
     @param proceed_without_rollback: Rollback on failure if False; otherwise, rollback.
+    @param reboot_device: If True, reboot device on success or failure, otherwise, do not reboot.
     """
 
     def __init__(self, trtl: Trtl, sota_cmd: str,  snap_num: Optional[str],
```

Comment:
Ensure that the `reboot_device` parameter is used consistently in all snapshotter classes. It seems that the `reboot_device` parameter is not being used in the `take_snapshot` method of the `WindowsSnapshot` and `DebianBasedSnapshot` classes. [important]

Reply:
We don't always want to use this variable.  It would not be used in the Windows scenario at this time.